Makefile

```diff
@@ -1,3 +1,11 @@
 .PHONY: build
 build:
-	go build -ldflags "-s -w -X 'github.com/friendsofgo/killgrave/internal/app/cmd._version=`git rev-parse --abbrev-ref HEAD`-`git rev-parse --short HEAD`'" -o bin/killgrave cmd/killgrave/main.go
\ No newline at end of file
+	go build -ldflags "-s -w -X 'github.com/friendsofgo/killgrave/internal/app/cmd._version=`git rev-parse --abbrev-ref HEAD`-`git rev-parse --short HEAD`'" -o bin/killgrave cmd/killgrave/main.go
+
+.PHONY: build-docker
+build-docker:
+	docker build --build-arg TAG=$(TAG) -t killgrave:$(TAG) .
```

Comment:
This doesn't work by default (e.g. `make build-docker`).

What about defining `TAG` in the Makefile itself? Like:

```
REF=$(shell git rev-parse --abbrev-ref HEAD)
HASH=$(shell git rev-parse --short HEAD)
TAG="${REF}-${HASH}"
```